contracts/deploy/515_configure_collator.js

```diff
@@ -36,17 +37,18 @@ module.exports = async function (hre) {
   const cfg = await ethers.getContract("CollatorConfig");
   const currentBlock = await ethers.provider.getBlockNumber();
   const activationBlockNumber = currentBlock + 10;
-
   const activeConfig = await cfg.getActiveConfig(activationBlockNumber);
   if (activeConfig[1].toNumber() === configSetIndex) {
     console.log("Collator config already added");
     return;
   }
 
-  await cfg.addNewCfg({
+  configSetIndex--;
```

Comment:
@ulope this "hack" seems to do the trick in the clean slate deployment case, but it's not correct